app/templates/overview/quick-start.hbs

```diff
@@ -0,0 +1,27 @@
+<h1>Quick start</h1>
+<p>All steps you need to make to start using Design System on your site.</p>
+
+<section>
+	<h2>1. Install with Bower</h2>
+	<p>As a first step, you need to install Design System libraries using Bower.</p>
+	<pre>bower install Wikia/design-system --save</pre>
+</section>
+
+<section>
+	<h2>2. Link to Design System css file</h2>
+	<p>In your html file link to styles.css file.</p>
+	<pre>
+&lt;head&gt;
+	...
+	&lt;link rel="stylesheet" type="text/css" href="bower_components/design-system/dist/css/styles.css"&gt;
+&lt;/head&gt;
+	</pre>
+</section>
+
+<section>
+	<h2>3. Start using Design System goodies!</h2>
+	<p>Now you can start using DS classes and variables in your application. To check if everything works, just add an example button and check if it is displayed correctly on your site.</p>
```

Comment:
If user includes CSS as we suggested above then she won't be able to use the variables.